tendrl/monitoring_integration/alert/handlers/cluster/gluster/brick_utilization_handler.py

```diff
@@ -0,0 +1,139 @@
+                    )
+                )
+                     
+            elif alert['severity'] == "INFO":
+                alert['tags']['message'] = (
+                    "Brick utilization of %s in node %s in "\
+                    "cluster %s is back normal" % (
+                        alert['tags']['brick_path'],
+                        alert['tags']['fqdn'],
+                        alert['tags']['cluster_name']
+                    )
+                )
+            else:
+                logger.log(
+                    "error",
+                    NS.publisher_id,
+                    {
+                        "message": "Alert %s have unsupported alert"
+                        "severity" % alert_json
+                    }
+                )
+                raise InvalidAlertSeverity 
+            return alert
+        except (KeyError,
+                CalledProcessError,
+                EtcdKeyNotFound,
+                NodeNotFound,
+                InvalidAlertSeverity) as ex:
+            Event(
+                ExceptionMessage(
+                    "error",
+                    NS.publisher_id, 
+                    {
+                        "message": "Error in converting grafana"
+                        "alert into tendrl alert %s" % alert_json,
+                        "exception": ex
+                    }
+                )
+            )
+    
+    def parse_alert_metrics(self, alert_json):
+        """
+        {
+          EvalData: {
+            evalMatches:[{
+              metric: "tendrl.clusters.ab3b125e-4769-4071-
+                      a349-e82b380c11f4.nodes.dhcp42-208_lab_eng_blr_
+                      redhat_com.bricks.|root|gluster_bricks
+                      |vol1_b2.utilization.percent-percent_bytes",
+              tags: null,
+              value: 15.614466017499998
+            }]
+          },
+          Settings: {
+            conditions: - [{
+              evaluator: - {
+                params: - [15],
+                type: "gt"
+              },
+              query : {
+                model : {
+                  target: "tendrl.clusters.ab3b125e-4769-
+                          4071-a349-e82b380c11f4.nodes.dhcp42-208_lab_eng_blr_
+                          redhat_com.bricks.|root|
+                          gluster_bricks|vol1_b2.utilization.percent-
+                          percent_bytes"
+                }
+              }
+            }]
+          }
+        }
+        """
+        alert = {}
+        alert['tags'] = {}
+        alert['current_value'] = utils.find_current_value(
+            alert_json['EvalData'])
+        target = utils.find_alert_target(
+            alert_json['Settings']['conditions'])
+        alert['tags']['plugin_instance'] = target
+        alert['tags']['warning_max'] = utils.find_warning_max(
+            alert_json['Settings']['conditions'][0]['evaluator']['params'])
+        metric = target.split(",")[0].split(".")
```

Comment:
better to hard code metric from handler
because not always this is going to be directly available from target especially for aggregated stuff.